### DISCUSSION TITLE
Are there plans for integrations with services like haveibeenpwned?

### DISCUSSION THREAD

**Discussion by Dich0tomy** [anonaddy/anonaddy#671] [Feature Requests]
As in the title; I would be inclined to pay additional couple bucks to be able for anonaddy to notify me when one of my aliases or "original" e-mails is detected in some of the pwned databases.

**Comment by willbrowningme** [comment]
It is on the [roadmap](https://github.com/anonaddy/anonaddy/issues/41). I do have a basic subscription with haveibeenpwned but in order to fully cover all aliases and domains I would need to upgrade to at least the "pwned 4" plan which is $1,370 per year, the plan after that being $2,740 per year.

I agree that it would be a nice addition to have so I'll see what I can do.

**Reply by Dich0tomy** [reply]
How would that change the price of the current plans?

**Reply by willbrowningme** [reply]
I don't have any plans to change the current plan prices.

**Reply by Dich0tomy** [reply]
But you're saying you will have to upgrade the plan, which will cost you more, therefore you will either lose any profit you had before from anonaddy or will have to pay from your own pocket. I'm not following.

**Reply by willbrowningme** [reply]
Yes addy.io would lose profit, I was simply making the point that this feature is a bigger commitment compared with others on the roadmap.

**Reply by Dich0tomy** [reply]
I see. Thanks a lot for the answer. Have a great day!
